.github/workflows/codeql.yml

```diff
@@ -29,5 +29,10 @@ jobs:
           languages: go
           queries: security-and-quality
 
+      - name: Setup go
```

Comment:
I fixed modifications referring to the following link. 
https://github.com/github/codeql/issues/15647#issuecomment-2003768106